CHANGELOG.md

```diff
@@ -19,6 +19,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Security
 
+## 0.1.2
+Released on July 21st, 2022.
+
+### Changed
+- Updated tests to be compatible with core Prefect library (v2.0b9) and bumped required version - [#30](https://github.com/PrefectHQ/prefect-gcp/pull/30)
+
```

Comment:
I don't think we need a new release for this. I think #31 will be a reason to release.

Reply:
Yeah, I was planning to get 31 inside here and release together